public/pages/techtalk.html

```diff
@@ -51,53 +51,66 @@
       <div class="">
-        <h2>Tech Talk: Are Code Bootcamps a fast-track to a career in tech?</h2>
-        <p>
-          Our Wednesday-night lab is online for now due to Covid! Join us February 9th from 6:30-8:00pm via Zoom using the link <a href="#zoom">below</a>!
-        </p>
-        <div style="float: left; margin: 0 10px; padding: 10px;float: left;text-align: center;">
-          <img src="../img/Erin-shaw.jpg" style="max-width; 150px;"/>
-          <h4 style="margin: 0; text-align: center;">Erin Shaw </h4>
-          <p style="margin-top: 5px;">Software Engineer @ Mailchimp<br>Bootcamp Graduate</p>
-        </div>
-        <p>
-          <a href="https://www.linkedin.com/in/erin-k-shaw/">Erin Shaw</a> is a software engineer at Mailchimp. She has been a programmer for almost five years. Before her career in tech, she was an archivist and corporate historian.
-         With a B.A. and M.A. in American History, she decided to attend The Grace Hopper Program at Fullstack Academy to lean full stack JavaScript in 2017. Afterwards, she was a participant at Recurse Center, a programming retreat located in New York City.
-        </p>
+          <h2>Tech Talk: Building a Data Analytics Platform Using Google Cloud</h2>
+          <p>
+            Our Wednesday-night lab is online for now due to Covid! Join us February 16th from 6:30-7:30pm via Zoom using the link <a href="#zoom">below</a>!
+          </p>
+          <p>
+             <a href="https://linkedin.com/in/mugizico"> JP Mugizi</a>
+          </p>
+          <p>
+            Jean-Paul is a Senior Software Engineer at Chipper Cash, a tech company  focusing on building  financial products for the African continent.
+            He specializes in data platform and distributed systems engineering.
+            Previously, he was an engineer at Mailchimp, Capital One, Login.gov / 18F, Boeing and others.
+            He was born and raised in Kigali City, Rwanda but currently resides in Brooklyn, NY.
+            Favorite hobbies include music, reading, cycling, and traveling.
+          </p>
+          <div style="left; margin: 0 10px; padding: 10px;left;text-align: center;">
+          <img src="../img/TechTalk-JP.jpg" style="max-width: 250px"/>
+          <h4 style="margin: 0; text-align: center;">JP Mugizi</h4>
+          <p style="margin-top: 5px;">Senior Software Engineer @ Chipper Cash<br>Specializes in Data Platform &
+            Distributed Systems Engineering.</p>
+          </div>
+
+
       </div>
 
       <!-- navigation-component 1.1 -->
       <aside class="aside-navigation small patch">
         <h4>Upcoming Talks</h4>
         <ul>
-          <li><a class="current">Feb. 9 - Erin Shaw: Are Code Bootcamps a fast-track to a career in tech?</a></li>
-          <li><a >Feb. 16 - JP Mugizi: Building a Data Analytics Platform Using Google Cloud</a></li>
+            <li>Feb. 16 - JP Mugizi: Building a Data Analytics Platform Using Google Cloud</a></li>
```

Comment:
By removing the `<a >` from this, it broke some styling that was applied to it. See:
![image](https://user-images.githubusercontent.com/1900802/153934669-7334530f-4592-44cf-a674-dc965f13fb6f.png)

It is no longer indented properly.

Reply:
Also, it needs to have `class="current"`

Reply:
The line should be:
```<li><a class="current">Feb. 16 - JP Mugizi: Building a Data Analytics Platform Using Google Cloud</a></li>```